docs/source/configuration.rst

```diff
@@ -46,11 +46,11 @@ Consider:
 
 .. code-block:: shell
 
-   tartufo --config myconfig.toml scan-local-repo .
+   tartufo --config myconfig.toml scan-local-repo <target_directory>
 
-``tartufo`` will look for ``myconfig.toml`` in the current directory, and then
+``tartufo`` will look for ``myconfig.toml`` in the target directory, and then
```

Comment:
This does not match what `--config` does, does it?

`--config` , if given a relative path, looks relative to the current directory, where you're running the command.

It also does not do some of the following things as far as I know, like looking in the parent directory. Though I think that logic is being removed in a separate PR regardless.

Reply:
Yes the --config seems to have removed the traversal logic and unused code is being removed as part of the other PR

Reply:
Okay, but it did not drastically change what `--config` is relative to, did it? It made `--config` relative to the target path instead of the current directory? That would be a huge change in behavior.

Reply:
You are right it looks at config relative to the current directory and not the target directory. I will revert the documentation here.